tests/feature_resolution_editions.js

```diff
@@ -423,16 +472,245 @@ tape.test("feature resolution editions precedence", function(test) {
         if (err)
             throw test.fail(err.message);
 
-    test.same(root.lookup("Message").lookupEnum("SomeEnumInMessage")._features,
-    {
-        enum_type: 'CLOSED',
-        field_presence: 'EXPLICIT',
-        json_format: 'LEGACY_BEST_EFFORT',
-        message_encoding: 'LENGTH_PREFIXED',
-        repeated_field_encoding: 'EXPANDED',
-        utf8_validation: 'NONE',
-        amazing_feature: 'G'
-      })
-    test.end();    
+        test.same(root.lookup("Message").lookupEnum("SomeEnumInMessage")._features,
```

Comment:
this part seems like the only other obvious breaking change but since we hadn't released features earlier seems ok. But just confirming the previous test wasn't right?

Reply:
Yea this was actually a bug fix :). The edition 2023 defaults weren't getting set correctly here, and proto2 defaults were being used